pvlib/ivtools.py

```diff
@@ -262,6 +262,173 @@ def fit_sde_sandia(voltage, current, v_oc=None, i_sc=None, v_mp_i_mp=None,
                                      v_oc)
 
 
+def fit_sdm_desoto(v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                   cells_in_series, EgRef=1.121, dEgdT=-0.0002677,
+                   temp_ref=25, irrad_ref=1000):
+    """
+    Calculates the parameters for the De Soto single diode model using the
+    procedure described in [1]. This procedure has the advantage of
+    using common specifications given by manufacturers in the
+    datasheets of PV modules. The solution is found using the
+    scipy.optimize.root() function, with the correpsonding default solver
+    method 'hybr'.
```

Comment:
It's probably worth mentioning that there are no restrictions on the fit variables, i.e., series resistance _could_ go negative.

Reply:
if you are touching this text, "correpsonding " is misspelled.